packages/amplify_authenticator/lib/src/state/inherited_forms.dart

```diff
@@ -19,12 +20,32 @@ class InheritedForms extends InheritedWidget {
 
   final ConfirmSignUpForm confirmSignUpForm;
 
-  final ConfirmSignInForm confirmSignInForm;
+  final ConfirmSignInMFAForm confirmSignInMFAForm;
+
+  final SendCodeForm sendCodeForm;
+
+  final ResetPasswordForm resetPasswordForm;
+
+  final ConfirmSignInNewPasswordForm confirmSignInNewPasswordForm;
 
+  //final ResetPasswordForm resetPasswordForm;
 
   // ignore: public_member_api_docs
-  static InheritedForms? of(BuildContext context) =>
-      context.dependOnInheritedWidgetOfExactType<InheritedForms>();
+  static InheritedForms? of(BuildContext context) {
```

Comment:
Super close - now just make this return `InheritedForms` (non-null) and remove all the `!` from the `InheritedForms.of(context)!....` calls.